### PR TITLE
sci-libs/calculix-ccx-2.8_p2 version bump

### PR DESCRIPTION
**Pull request by Fat-Zer**
Mostly fix issue #420. 
Somebody still have to do something about the missing sources of cxx-2.7.

### CODE REVIEW

File: sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2014 Gentoo Foundation
```
Comment (marbre) [inline sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild]: Please fix the year.

File: sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	export CC="$(tc-getCC)"
+	export FC="$(tc-getFC)"
+}
+
+src_install () {
+	dobin ${MY_P}
+	dosym ${MY_P} /usr/bin/ccx
+
+	if use doc; then
+		cd "${S}/../doc"
```
Comment (marbre) [inline sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild]: Missing die.

File: sci-libs/calculix-ccx/files/01_ccx_2.8p2_Makefile_custom_cc_flags_spooles_arpack.patch [hunk 3]
```diff
@@ -0,0 +1,31 @@
+diff -ru CalculiX.orig/ccx_2.8p2/src/Makefile CalculiX/ccx_2.8p2/src/Makefile
```
Comment (marbre) [inline sci-libs/calculix-ccx/files/01_ccx_2.8p2_Makefile_custom_cc_flags_spooles_arpack.patch]: Please clean up the patch and add a short description like explained in http://dev.gentoo.org/~vapier/clean-patches

**Comment by Fat-Zer** [comment]
All done.

File: sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+	if use threads; then
+		append-cflags "-DUSE_MT"
+	fi
+
+	if use arpack; then
+		export ARPACKLIB=$($(tc-getPKG_CONFIG) --libs arpack)
+		append-cflags "-DARPACK"
+	fi
+	export CC="$(tc-getCC)"
+	export FC="$(tc-getFC)"
```
Comment (jlec) [inline sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild]: If the build uses fortran, you need to use the fortran.eclass.

Comment (marbre) [inline sci-libs/calculix-ccx/calculix-ccx-2.8_p2.ebuild]: Sorry, I've noticed that. @Fat-Zer Can you take a look into that?

**Comment by Fat-Zer** [comment]
@jlec, yep you seems no be right. 
Thanks for notice, haven't knew that.

**Pull request merged**